### PR TITLE
Wt 303 link social accounts

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Added link and unlink options to the security profile section
- Notes: [We still need to introduce error messages](https://dailydotdev.atlassian.net/browse/WT-305) (think only if the account is already existing somewhere) (separate ticket)
- Open question: Do we want to create a useSettings hook? (might be confusing as it's basically only Kratos settings, not global settings)
- Notes 2: We have no way of knowing which providers are already linked or not, so that's something we have to refactor based on whoami changes. (boot)

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-303 #done

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
LGTM! 🚢 🚢 🚢

**Comment by sshanzel** [comment]
I can help with merging the conflicts here, just let me know.

**Comment by rebelchris** [comment]
@sshanzel This one is still blocked because we don't have the endpoint, so i'll sort them out, thanks for the offer 🙏

**Comment by rebelchris** [comment]
@sshanzel Migrated, I think we can merge this so long.
We need to fix the TODO mentioned in the code with the new popup for the privileged actions

File: packages/webapp/components/layouts/AccountLayout/Security/index.tsx [hunk 1]
```diff
@@ -24,14 +34,56 @@ interface AccountSecurityDefaultProps {
   isEmailSent?: boolean;
   onSwitchDisplay: (display: Display) => void;
 }
+export interface ManageSocialProvidersProps {
+  type: 'link' | 'unlink';
```
Comment (sshanzel) [inline packages/webapp/components/layouts/AccountLayout/Security/index.tsx:38]: I think we can create a dedicated type for this and export it so the component requiring it as props would also be type-strict.

**Review by sshanzel — APPROVED** [review]
Just one thing about type-safety. Can't wait to test this out! Amazing! 🚢

**Pull request merged**